configs/crowddet/crowddet_r50_fpn_8xb2-30e_crowdhuman.py

```diff
@@ -0,0 +1,207 @@
+_base_ = ['../_base_/default_runtime.py']
```

Comment:
Is `crowddet_r50_fpn_8xb2-30e_crowdhuman.py` appropriately named? If `retinanet` is supported in the future, how will it be prescribed?

Reply:
`crowddet-rcnn_r50_fpn_8xb2-30e_crowdhuman.py`